terraform/s3.tf

```diff
@@ -0,0 +1,80 @@
+## Rehydrate S3 Bucket ##
+resource "aws_s3_bucket" "rehydration_s3_bucket" {
+  bucket = local.rehydration_bucket_name
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  tags = merge(
+    local.common_tags,
+    {
+      "Name"         = "pennsieve-${var.environment_name}-rehydration-s3-bucket-${data.terraform_remote_state.region.outputs.aws_region_shortname}"
+      "name"         = "pennsieve-${var.environment_name}-rehydration-s3-bucket-${data.terraform_remote_state.region.outputs.aws_region_shortname}"
+      "service_name" = "rehydration"
+      "tier"         = "s3"
+    },
+  )
+}
+
+resource "aws_s3_bucket_cors_configuration" "rehydration_s3_bucket_cors" {
+  bucket = aws_s3_bucket.rehydration_s3_bucket.bucket
+
+  cors_rule {
+    allowed_headers = ["*"]
+    allowed_methods = ["GET", "HEAD"]
+    allowed_origins = ["*"]
+    max_age_seconds = 3000
+  }
+}
+
+resource "aws_s3_bucket_acl" "rehydration_s3_acl" {
+  bucket     = aws_s3_bucket.rehydration_s3_bucket.id
+  acl        = "public-read"
+  depends_on = [aws_s3_bucket_ownership_controls.rehydration_s3_bucket_ownership_controls]
+}
+
+resource "aws_s3_bucket_ownership_controls" "rehydration_s3_bucket_ownership_controls" {
+  bucket = aws_s3_bucket.rehydration_s3_bucket.id
+  rule {
+    object_ownership = "ObjectWriter"
```

Comment:
What's the motivation for using Object Writer here, rather than enforcing Bucket Owner?

Reply:
Thanks for pointing this and the ACL thing out @muftring. Both were pure copy and paste, so no real motivation. I guess they are linked in that you can now only have ACLs if you don't enforce Bucket Owner. 

It looks like the bucket policy and the block public access parts already make this a public bucket, so I'm just going to delete the ACL and ownership controls stuff.